Counterexamples/EulerSumOfPowers.lean

```diff
@@ -136,7 +136,7 @@ theorem existsEqualSumsOfLikePowersFor_of_sumOfPowersConjectureWith (R : Type*)
 After the first counterexample was found, Leon J. Lander, Thomas R. Parkin, and John Selfridge
 made a similar conjecture that is not amenable to the counterexamples found so far.
 The status of this conjecture is unknown.
-https://en.wikipedia.org/wiki/Lander,_Parkin,_and_Selfridge_conjecture
+https://en.wikipedia.org/wiki/Lander%2C_Parkin%2C_and_Selfridge_conjecture
```

Comment:
I think a better answer might be to write `<https://en.wikipedia.org/wiki/Lander,_Parkin,_and_Selfridge_conjecture>` which is the markdown way of saying "this is definitely a link", or `[See Wikipedia](https://en.wikipedia.org/wiki/Lander,_Parkin,_and_Selfridge_conjecture)`, since the doc-gen reader almost certainly does not care to see the URL.